src/test/java/com/google/cloud/compute/v1/it/ITComputeTest.java

```diff
@@ -51,41 +262,951 @@
+    addressClient.close();
+    autoscalerClient.close();
+    backendBucketClient.deleteBackendBucket(PROJECT_BACKEND_BUCKET_NAME);
+    backendBucketClient.close();
+    imageClient.deleteImage(PROJECT_GLOBAL_IMAGE_NAME);
+    imageClient.close();
+    Thread.sleep(30000);
+    diskClient.deleteDisk(DISK_SELF_LINK);
+    diskClient.close();
     diskTypeClient.close();
+    externalVpnGatewayClient.close();
+    firewallClient.deleteFirewall(FIREWALL);
+    firewallClient.close();
+    forwardingRuleClient.deleteForwardingRule(PROJECT_REGION_FORWARDING_RULE_NAME);
+    forwardingRuleClient.close();
+    globalAddressClient.close();
+    globalForwardingRuleClient.close();
+    globalOperationClient.close();
+    healthCheckClient.close();
+    httpHealthCheckClient.deleteHttpHealthCheck(HTTP_HEALTH_CHECK_NAME);
+    httpHealthCheckClient.close();
+    httpsHealthCheckClient.deleteHttpsHealthCheck(HTTPS_HEALTH_CHECK_NAME);
+    httpsHealthCheckClient.close();
     instanceClient.close();
+    instanceGroupClient.deleteInstanceGroup(INSTANCE_GROUP_NAME);
+    instanceGroupClient.close();
+    instanceTemplateClient.close();
+    instanceGroupManagerClient.close();
+    interconnectAttachmentClient.close();
+    interconnectLocationClient.close();
+    interconnectClient.close();
+    licenseClient.deleteLicense(LICENSE_NAME);
+    licenseClient.close();
+    machineTypeClient.close();
+    networkEndpointGroupClient.deleteNetworkEndpointGroup(NETWORK_ENDPOINT_GROUP_NAME);
+    networkEndpointGroupClient.close();
+    networkClient.close();
+    nodeTemplateClient.close();
+    nodeTypeClient.close();
+    nodeGroupClient.close();
+    projectClient.close();
+    regionAutoscalerClient.close();
+    regionClient.close();
+    regionCommitmentClient.close();
+    regionDiskClient.close();
+    regionBackendServiceClient.close();
+    targetHttpProxyClient.close();
+    targetHttpsProxyClient.close();
+    targetInstanceClient.close();
+    Thread.sleep(30000);
+    targetPoolClient.deleteTargetPool(REGION_TARGET_POOL_NAME);
+    targetPoolClient.close();
+    targetSslProxyClient.close();
+    targetTcpProxyClient.close();
+    targetVpnGatewayClient.close();
+    urlMapClient.close();
+    vpnGatewayClient.close();
+    vpnTunnelClient.close();
+    zoneClient.close();
+    if (storage != null) {
+      boolean wasDeleted = RemoteStorageHelper.forceDelete(storage, BUCKET, 10, TimeUnit.SECONDS);
+      if (!wasDeleted && LOG.isLoggable(Level.WARNING)) {
+        LOG.log(Level.WARNING, "Deletion of bucket {0} timed out, bucket is not empty", BUCKET);
+      }
+    }
+  }
+
+  @Test
+  public void listAcceleratorTypesTest() {
+    AcceleratorTypeClient.ListAcceleratorTypesPagedResponse listAcceleratorTypes =
+        acceleratorTypeClient.listAcceleratorTypes(PROJECT_ZONE_NAME);
+    List<AcceleratorType> acceleratorTypes = Lists.newArrayList(listAcceleratorTypes.iterateAll());
+    assertThat(acceleratorTypes).isNotNull();
```

Comment:
here and in other places you do the following:
```
List<SomeType> lst = Lists.newArrayList(...);
assertThat(lst).isNotNull();
```
But Lists.newArrayList(Iterable) will never return null:[ Lists.java](https://github.com/google/guava/blob/master/guava/src/com/google/common/collect/Lists.java)
So, such checks are redundant.

It will be more efficient to verify that 

```
assertNotNull(listAcceleratorTypes);
assertThat(listAcceleratorTypes.size() > 0);
```

Reply:
done

Reply:
done